.github/workflows/sycl-linux-precommit.yml

```diff
@@ -51,9 +51,6 @@ jobs:
     name: Decide which Arc tests to run
     needs: [build, detect_changes]
     if: ${{ always() && !cancelled() && needs.build.outputs.build_conclusion == 'success' }}
-    # Github's ubuntu-* runners are slow to allocate. Use our CUDA runner since
-    # we don't use it for anything right now.
-    runs-on: [Linux, build]
```

Comment:
``

The workflow is not valid. .github/workflows/sycl-linux-precommit.yml (Line: 51, Col: 5): Required property is missing: runs-on
```

@aelovikov-intel What am I supposed to put here? Is it ubuntu-latest or something?